job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy

```diff
@@ -31,13 +31,25 @@ pipeline{
                 ]
             }
         }
+        
+        stage('trigger nightly job 7.59.x') {
+            steps {
+                build job: 'nightly/7.59.x', propagate: false, wait: true, parameters: [
+                        [\$class: 'StringParameterValue', name: 'KIE_GROUP_DEPLOYMENT_REPO_URL', value: 'https://bxms-qe.rhev-ci-vms.eng.rdu2.redhat.com:8443/nexus/service/local/repositories/scratch-release-rhba-7.12/content-compressed'],
+                        [\$class: 'StringParameterValue', name: 'UMB_VERSION', value: '712'],
+                        [\$class: 'StringParameterValue', name: 'PRODUCT_VERSION', value: '7.12.0'],
+                        [\$class: 'StringParameterValue', name: 'DEFAULT_CONFIG_BRANCH', value: '7.59.x'],
+                        [\$class: 'BooleanParameterValue', name: 'SKIP_TESTS', value: true]
+                ]
+            }
+        }
 
         // Kogito prod nightlies
         stage('trigger kogito nightly job main') {
             steps {
                 build job: 'kogito.nightly/main', propagate: false, wait: true, parameters: [
                         [\$class: 'StringParameterValue', name: 'RHBA_MAVEN_REPO_URL', value: 'http://bxms-qe.rhev-ci-vms.eng.rdu2.redhat.com:8081/nexus/content/repositories/rhba-main-nightly-with-upstream'],
-                        [\$class: 'StringParameterValue', name: 'RHBA_VERSION_PREFIX', value: "${RHBA_VERSION_PREFIX}"],
+                        [\$class: 'StringParameterValue', name: 'RHBA_VERSION_PREFIX', value: "7.59.1.redhat-"],
```

Comment:
this should be matching the version from main of https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/main/pom.xml#L25 -> 7.60.0-SNAPSHOT translate into 7.60.0.Final or later to 7.61.0.Final etc, so it is not good to use hardcoded string. Better to keep variable and fix the job parameter or take/create it from `version.kie` property.